### PR TITLE
Update product fixture to grab SQLProduct

### PR DESCRIPTION
**Pull request by proteusvacuum**
Bulk saving products was not updating the last_modified date. 
I changed this to pull directly from SQLProduct too, since it otherwise it fetches from SQL then couch, and the data is the same. 

http://manage.dimagi.com/default.asp?225618

@sravfeyn FYI: @phillipm @esoergel

### CODE REVIEW

File: corehq/apps/products/models.py [hunk 1]
```diff
@@ -60,6 +60,7 @@ def get_codes(domain):
             return codes_by_domain[domain]
 
         for doc in docs:
+            doc.last_modified = datetime.utcnow()
```
Comment (esoergel) [inline corehq/apps/products/models.py]: :+1:

**Comment by esoergel** [comment]
:+1: on :construction_worker:

**Comment by esoergel** [comment]
Test failure looks real, my guess is it's just a difference between the way couch and sql products render to XML

```
======================================================================
FAIL: test_product_fixture (corehq.apps.commtrack.tests.test_fixture.FixtureTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/corehq/apps/commtrack/tests/test_fixture.py", line 119, in test_product_fixture
    ElementTree.tostring(fixture[0])
  File "/mnt/corehq/apps/app_manager/tests/util.py", line 32, in assertXmlEqual
    _check_shared(expected, actual, LXMLOutputChecker(), "xml")
  File "/mnt/corehq/apps/app_manager/tests/util.py", line 126, in _check_shared
    raise AssertionError(message)
AssertionError: XML mismatch
```

Specifically, `cost` and `unit` vary.  The `cost` difference I'd chalk up to type `Decimal` idiosyncracies, but the missing `unit`s could be something more real, like bad sync code.
```
-      <unit/>
+      <unit>icfwaretcbrrcayijniv</unit>
-      <cost>14.85720</cost>
+      <cost>14.8572</cost>
 ```

**Comment by proteusvacuum** [comment]
I think out of scope to switch this to `SQLProduct`, so reverted to `Product`

File: corehq/apps/products/models.py [hunk 2]
```diff
@@ -329,5 +330,9 @@ def __repr__(self):
     def by_domain(cls, domain):
         return cls.objects.filter(domain=domain).all()
 
+    @property
+    def get_id(self):
```
Comment (czue) [inline corehq/apps/products/models.py]: nit: don't think this is strictly necessary anymore

**Pull request merged**